haystack_experimental/core/pipeline/pipeline.py

```diff
@@ -566,6 +571,15 @@ def save_state(
 
         dt = datetime.now()
         file_name = Path(f"{component_name}_{dt.strftime('%Y_%m_%d_%H_%M_%S')}.json")
+        # we store init_parameters for the component with breakpoint in the saved state
```

Comment:
I was a bit confused with some of the comments

```suggestion
        # we store a component init_parameters together with breakpoint in the saved state
```